lib/zendesk_apps_support/validations/requirements.rb

```diff
@@ -92,23 +106,22 @@ def invalid_channel_integrations(requirements)
           end
         end
 
-        def invalid_custom_resources_schema(requirements)
-          custom_resources_schema = requirements['custom_resources_schema']
-          return unless custom_resources_schema
-          valid_schema_keys = %w[resource_types relationship_types]
+        def invalid_custom_objects(requirements)
+          custom_objects = requirements['custom_objects']
+          return unless custom_objects
+
+          valid_schema = {
+            'custom_object_types' => %w[key schema],
```

Comment:
`end_users_can_read` is an optional writable key

Reply:
Thanks, good catch! I've removed the check for extra keys.